doc/man/8/radosgw-admin.rst

```diff
@@ -135,7 +135,8 @@ as follows:
   Purge bucket index entries.
 
 :command:`object rm`
-  Remove an object.
+  Remove an object. Include "--yes-i-really-mean-it" to remove object's
```

Comment:
Suggest 

>Remove an S3 / Swift object

while implicit here I’ve been trying to disambiguate RADOS and RGW objects

Below suggest
> entry from the bucket index, for example if it is damaged.